### ISSUE TITLE
Update descriptions for ecosystems page

### ISSUE THREAD

**Issue by EssamWisam** [JuliaAI/MLJ#5]
It was nontrivial to maintain support for markdown after replacing the modal with the hovering tooltip as can be [seen here](https://mlj.netlify.app/ecosystem) and there is no purpose for links anyway. Will need to change the file in https://github.com/JuliaAI/MLJ/blob/main/src/data/EcosystemPage.yaml .

CC @ablaom

**Comment by ablaom** [comment]
So you just want me to unmarkdown-ify the `orgMarkdownDesc`?

**Comment by EssamWisam** [comment]
Yes along with improving any description in case its too long (as can be evaluated by hovering on them in the sire) or if the description requires or strongly assumes that the user clicks on some link.

**Comment by EssamWisam** [comment]
@ablaom Do let me know if you need help here or in the missing part of the homepage (get started). Is there anything less needed before the conclusion of this project (aside from hosting and README)?

**Comment by EssamWisam** [comment]
Gentle reminder.
